chatlearn/tools/megatron_checkpoint_utils.py

```diff
@@ -263,13 +263,13 @@ def load_module(self, name):
 
 if __name__ == '__main__':
     if exist_checkpoint_util():
-        sys.meta_path.insert(-1, CheckpointUtilsImporter('tools.checkpoint.util', \
+        sys.meta_path.insert(0, CheckpointUtilsImporter('tools.checkpoint.util', \
             'tools.checkpoint.loader_megatron', 'tools.checkpoint.saver_megatron'))
         from tools.checkpoint import loader_megatron, saver_megatron # pylint: disable=unused-import
         from tools.checkpoint import util
         util.main()
     else:
-        sys.meta_path.insert(-1, CheckpointUtilsImporter('tools.checkpoint.convert', \
+        sys.meta_path.insert(0, CheckpointUtilsImporter('tools.checkpoint.convert', \
```

Comment:
We could remove the CheckpointUtilsImporter from sys.meta_path after convert is compiled.

Reply:
Ack, Thx.